src/index.js

```diff
@@ -57,3 +58,6 @@ export {default as ContinuousSizeLegend}
   from './lib/legends/continuous-size-legend';
 
 export {default as makeWidthFlexible} from './lib/make-vis-flexible';
+
+export {default as AxisUtils} from './lib/utils/axis-utils';
+export {default as ScalesUtils} from './lib/utils/scales-utils';
```

Comment:
For consistency, how about exporting this as `ScaleUtils`? Since `Axis` is singular